.github/workflows/python-app-ci.yml

```diff
@@ -65,3 +65,6 @@ jobs:
           pytest --cov=./ --cov-report=xml
       - name: Upload
         uses: codecov/codecov-action@v3
+      - name: Run tests with coverage
+        run: |
+          pytest --cov=./src/verifier/core --cov-report=term-missing --cov-fail-under=50
```

Comment:
This looks good for now. Thank you for pointing out that we don't have tests outside of core.